src/Hooks/Form.md

```diff
@@ -17,17 +17,33 @@ window.esForms.init();
 This will manually re-init all JS for the forms, you can use this option in combination with the global setting to disable auto-init of JS on page load.
 
 ```js
-window.addEventListener('load', function () {
+function initForms() {
 	if (!window?.esForms) {
```

Comment:
Can we guarantee that our `esForms` property is available in the window object before we come to this point?
(Plugin code before our theme code?)